packages/story-editor/src/components/workspaceFooter/index.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { default } from './workspaceFooter';
```

Comment:
I considered moving the other elements of the workspace footer inside the folder as subfolders alongside the carousel and the grid view.

These other items are the checklist and the keyboard shortcuts menu. While the latter is pretty small, the former is a rather big component and it seemed like a big change, so didn't do either.

Ping @BrittanyIRL for thoughts.

Reply:
I can go either way. They are part of the footer but as far as finding files go, don't think it hurts having them kept at the components/ level.

Reply:
I'll leave it as-is for now.